monai/transforms/intensity/array.py

```diff
@@ -855,7 +1063,13 @@ def __init__(self, prob: float = 0.1, gamma: Sequence[float] | float = (0.5, 4.5
         else:
             self.gamma = (min(gamma), max(gamma))
 
-        self.gamma_value: float | None = None
+        self.gamma_value: float = 1.0
+        self.invert_image: bool = invert_image
+        self.retain_stats: bool = retain_stats
+
+        self.adjustContrast = AdjustContrast(
```

Comment:
```suggestion
        self.adjust_contrast = AdjustContrast(
```
Unless we have good reason otherwise, we should stick to pep8 snake-case names here and elsewhere.

Reply:
Thanks, I changed this according to your suggestion.